prost-build/src/lib.rs

```diff
@@ -252,6 +253,14 @@ impl Config {
         self
     }
 
+    /// Configure the code generator to use btree_map for all protobuf map types
+    /// and never HashMap. This is useful if the generated code must be no_std.
+    pub fn force_btree_map(&mut self) -> &mut Self {
```

Comment:
This can already be done with `btree_map`, I'd prefer not to have another option to support.

Reply:
i didn't realize this, thanks